src/adapters/supabase/helpers/client.ts

```diff
@@ -90,6 +90,7 @@ export type UserProfileAdditions = {
 };
 const getDbDataFromUserProfile = (userProfile: UserProfile, additions?: UserProfileAdditions) => {
   return {
+    user_id: userProfile.id,
```

Comment:
you need to also add this field to `upsertUser`

Reply:
Done, will be implemented in next commit.